gulpfile.js

```diff
@@ -103,7 +103,7 @@ gulp.task('common', ['third_party'], function() {
 
 var closureCompilerConfig = {
   compilerPath: 'bin/closure-compiler.jar',
-  fileName: 'build/perfkit_scripts.js',
+  fileName: 'deploy/client/perfkit_scripts.js',
```

Comment:
Should be able to re-base to eliminate this local change.  I've ported it from master to dev-cloudsql already.